lib/SyTest/Homeserver/Dendrite.pm

```diff
@@ -101,6 +101,7 @@ sub _get_config
          server_name => $self->server_name,
          private_key => $self->{paths}{matrix_key},
          presence_enabled => $JSON::true,
+         http_client_tls_insecure_skip_verify_for_testing => $JSON::true,
```

Comment:
This is in the wrong place. It's in global here but the dendrite pr adds this flag to the pushers config.

Reply:
Yepp, revised.